test/integration/multikueue/suite_test.go

```diff
@@ -131,7 +132,12 @@ func managerSetup(ctx context.Context, mgr manager.Manager) {
 	failedCtrl, err := core.SetupControllers(mgr, queues, cCache, configuration)
 	gomega.Expect(err).ToNot(gomega.HaveOccurred(), "controller", failedCtrl)
 
-	failedWebhook, err := webhooks.Setup(mgr, ptr.Deref(configuration.MultiKueue.DispatcherName, config.MultiKueueDispatcherModeAllAtOnce))
+	// allow overriding dispatcher name via context value, fallback to configuration default
+	dispatcherName, _ := ctx.Value(dispatcherCtxKey{}).(string)
+	if dispatcherName == "" {
+		dispatcherName = ptr.Deref(configuration.MultiKueue.DispatcherName, config.MultiKueueDispatcherModeAllAtOnce)
+	}
+	failedWebhook, err := webhooks.Setup(mgr, dispatcherName)
```

Comment:
Hm, we would not need to do it, if we just eliminated the parameter (as we plan to do anyway). 

What about combining all PRs into one: fix for elastic Jobs and MultiKueue cleanup to drop the param? Could we then just cherrypick them together easier?